src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-wrapper">
+      <div class="container">
+        <div class="calendar calendar--month-length--31">
+          <div class="calendar__day calendar__day--start-day--sat"></div>
```

Comment:
check task description: start-day should be a modifier for calendar (not for calendar__day)